lib/feature/home_page/widgets/pokemon_tile.dart

```diff
@@ -49,57 +52,62 @@ class _PokemonTileState extends State<PokemonTile> {
     return GestureDetector(
-      child: Padding(
-        padding: EdgeInsets.fromLTRB(10, 0, 10, 0),
-        child: Container(
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(10),
-            color: Colors.white,
-            boxShadow: [
-              BoxShadow(
-                color: Colors.grey.withOpacity(0.5),
-                spreadRadius: 5,
-                blurRadius: 7,
-                offset: const Offset(0, 3), // changes position of shadow
-              ),
-            ],
-          ),
-          child: Column(children: [
-            const SizedBox(height: 8),
-            Text(
-              widget.thisPokemon.name?.capitalize ?? '',
-              style: const TextStyle(
-                color: Colors.black,
-                fontWeight: FontWeight.bold,
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(15),
+          color: thisTileTypes?.subTypes?.first.type?.name!.getPokemonColor ?? Colors.white,
+        ),
+        child: Stack(
+          children: [
+            Positioned(
+              left: 5,
+              top: 5,
+              child: Text(
+                widget.thisPokemon.name?.capitalize ?? '',
+                style: const TextStyle(
+                  color: Colors.white,
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20,
+                ),
               ),
-              textAlign: TextAlign.left,
             ),
-            const SizedBox(height: 15),
-            Row(
-              children: [
-                Padding(
-                  padding: const EdgeInsets.fromLTRB(20, 1, 1, 1),
-                  child: Column(children: [
-                    Text('${thisTileTypes?.subTypes?.first.type?.name ?? ''}'),
-                    Text(
-                        '${thisTileTypes?.subTypes?.last.type?.name == thisTileTypes?.subTypes?.first.type?.name ? '' : thisTileTypes?.subTypes?.last.type?.name}'),
-                  ]),
-                ),
-                Container(
-                  width: 100,
-                  height: 100,
-                  alignment: Alignment.centerRight,
-                  child: Image.network(
-                    '$pokemonImgUrl${thisTileTypes?.id}.png',
-                    fit: BoxFit.cover,
-                    errorBuilder: (_, __, ___) {
-                      return CircularProgressIndicator();
-                    },
+            Positioned(
+              left: 5,
+              top: 35,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  PillContainerWidget(
+                    type: thisTileTypes?.subTypes?.first.type?.name,
+                    color: typeDetailsPageBackgroundColor,
                   ),
-                )
-              ],
+                  SizedBox(height: 5),
+                  Visibility(
```

Comment:
instead of using visibility. Add an if statement
`if(condition)`
`  PillContainterWidget()`